Tools/Pipeline/Common/PipelineTypes.cs

```diff
@@ -408,6 +408,9 @@ private static void ResolveAssemblies(IEnumerable<string> assemblyPaths)
                 try
 #endif
                 {
+                    if (!asm.ToString().Contains("MonoGame"))
+                        continue;
```

Comment:
No, it won't fail because if you expand above and bellow you'll notice that that part is used for scanning referenced assemblies to the Pipeline tool, not referenced assemblies for the .mgcb file.

https://github.com/cra0zy/MonoGame/blob/ed1/Tools/Pipeline/Common/PipelineTypes.cs#L403